### PR TITLE
Fixed azure ad related docs for idp groups import setup

### PR DESCRIPTION
**Pull request by rujhan-arora-astronomer**
This PR adds some missing info related to azure ad for IDP groups import setup

### CODE REVIEW

**Review by rishkarajgi — APPROVED** [review]
lgtm

File: software/integrate-auth-system.md [hunk 1]
```diff
@@ -114,9 +114,11 @@ Follow these steps to configure Azure AD as your OIDC provider.
   
     - **Name**: Any
     - **Supported account types**: Accounts in this organizational directory only (Astronomer only - single tenant)
-    - **Redirect URI**: Web / `https://houston.BASEDOMAIN/v1/oauth/redirect/`.
+    - **Redirect URIs**:
+        - Web / `https://houston.BASEDOMAIN/v1/oauth/redirect/`.
+        - Web / `https://houston.BASEDOMAIN/v1/oauth/callback/`.
 
-    Replace `BASEDOMAIN` with your own. For example, if your base domain is `mycompany.com`, your redirect URI is https://houston.mycompany.com/v1/oauth/redirect/
+    Replace `BASEDOMAIN` with your own. For example, if your base domain is `mycompany.com`, your redirect URIs should be https://houston.mycompany.com/v1/oauth/redirect/ and https://houston.mycompany.com/v1/oauth/callback/.
```
Comment (jwitz) [inline software/integrate-auth-system.md:121]:
```suggestion
    Replace `BASEDOMAIN` with your own. For example, if your base domain is `mycompany.com`, your redirect URIs should be `https://houston.mycompany.com/v1/oauth/redirect/` and `https://houston.mycompany.com/v1/oauth/callback/`.
```

File: software/integrate-auth-system.md [hunk 2]
```diff
@@ -139,22 +141,25 @@ Complete this setup only if you want to import Azure AD groups to Astronomer Sof
...
+7. Click **Microsoft Graph** and add the following minimum permissions for Microsoft Graph:
 
     - `email`
     - `Group.Read.All`
     - `openid`
     - `profile`
-    - `User.Read`
+    - `User.Read
+    `<br /> **Note:** Make sure that you have granted the Admin Consent for all these permissions.
+    ![azure_api_permissions_consent.png](/img/software/azure_api_permissions_consent.png)
```
Comment (jwitz) [inline software/integrate-auth-system.md:152]:
```suggestion
    
    For each of these permissions, select **Grant Admin Consent for Astronomer Data**. Your Microsoft Graph permissions should look similar to the following image:
    
    ![Completed permissions page in Azure](/img/software/azure_api_permissions_consent.png)
```

File: software_versioned_docs/version-0.31/integrate-auth-system.md [hunk 3]
```diff
@@ -114,9 +114,11 @@ Follow these steps to configure Azure AD as your OIDC provider.
   
     - **Name**: Any
     - **Supported account types**: Accounts in this organizational directory only (Astronomer only - single tenant)
-    - **Redirect URI**: Web / `https://houston.BASEDOMAIN/v1/oauth/redirect/`.
+    - **Redirect URIs**:
+        - Web / `https://houston.BASEDOMAIN/v1/oauth/redirect/`.
+        - Web / `https://houston.BASEDOMAIN/v1/oauth/callback/`.
 
-    Replace `BASEDOMAIN` with your own. For example, if your base domain is `mycompany.com`, your redirect URI is https://houston.mycompany.com/v1/oauth/redirect/
+    Replace `BASEDOMAIN` with your own. For example, if your base domain is `mycompany.com`, your redirect URIs should be https://houston.mycompany.com/v1/oauth/redirect/ and https://houston.mycompany.com/v1/oauth/callback/.
```
Comment (jwitz) [inline software_versioned_docs/version-0.31/integrate-auth-system.md:121]:
```suggestion
    Replace `BASEDOMAIN` with your own. For example, if your base domain is `mycompany.com`, your redirect URIs should be `https://houston.mycompany.com/v1/oauth/redirect/` and `https://houston.mycompany.com/v1/oauth/callback/`.
```

File: software_versioned_docs/version-0.31/integrate-auth-system.md [hunk 4]
```diff
@@ -139,22 +141,25 @@ Complete this setup only if you want to import Azure AD groups to Astronomer Sof
...
+7. Click **Microsoft Graph** and add the following minimum permissions for Microsoft Graph:
 
     - `email`
     - `Group.Read.All`
     - `openid`
     - `profile`
-    - `User.Read`
+    - `User.Read
+    `<br /> **Note:** Make sure that you have granted the Admin Consent for all these permissions.
+    ![azure_api_permissions_consent.png](/img/software/azure_api_permissions_consent.png)
```
Comment (jwitz) [inline software_versioned_docs/version-0.31/integrate-auth-system.md:152]:
```suggestion
    
    For each of these permissions, select **Grant Admin Consent for Astronomer Data**. Your Microsoft Graph permissions should look similar to the following image:
    
    ![Completed permissions page in Azure](/img/software/azure_api_permissions_consent.png)
```

File: software/integrate-auth-system.md (hunk 1 above)
Comment (jwitz) [inline software/integrate-auth-system.md:121]:
```suggestion
    Replace `BASEDOMAIN` with your own. For example, if your base domain is `mycompany.com`, your redirect URIs should be `https://houston.mycompany.com/v1/oauth/redirect/` and `https://houston.mycompany.com/v1/oauth/callback/`.
```

**Review by jwitz — APPROVED** [review]
Looks great @rujhan-arora-astronomer ! I am just making some minor copy changes to the content and will merge this. Thank you for getting this fixed.

File: software/integrate-auth-system.md [hunk 5]
```diff
@@ -139,22 +141,27 @@ Complete this setup only if you want to import Azure AD groups to Astronomer Sof
...
 6. Click **API permissions** in the left menu.
-5. Click **Microsoft Graph** and add the following minimum permissions for Microsoft Graph:
+7. Click **Microsoft Graph** and add the following minimum permissions for Microsoft Graph:
 
     - `email`
     - `Group.Read.All`
     - `openid`
     - `profile`
-    - `User.Read`
+    - `User.Read
```
Comment (jwitz) [inline software/integrate-auth-system.md:150]:
```suggestion
    - `User.Read`
```

**Review by jwitz — APPROVED** [review]

**Pull request merged**